package.json

```diff
@@ -26,26 +26,26 @@
   },
   "dependencies": {
     "asar-require": "0.2.0",
-    "optimist": "~0.4.0",
     "async": "~0.2.8",
-    "underscore-plus": "1.x",
-    "temp": "~0.7.0",
-    "wrench": "~1.5.1",
-    "ncp": "~0.5.1",
-    "npm": "2.5.1",
-    "season": ">=5.0.5 <6.0",
     "colors": "~0.6.1",
-    "wordwrap": "0.0.2",
-    "fs-plus": "2.x",
-    "keytar": ">=2.1 <3",
-    "read": "~1.0.5",
-    "q": "~0.9.7",
-    "plist": "git+https://github.com/nathansobo/node-plist.git",
+    "exit": "0.1.2",
     "first-mate": ">=3 <4",
+    "fs-plus": "2.x",
     "git-utils": ">=3 <4",
+    "keytar": ">=2.1 <3",
     "mv": "2.0.0",
+    "ncp": "~0.5.1",
+    "npm": "2.5.1",
     "open": "0.0.4",
-    "exit": "0.1.2"
+    "plist": "git+https://github.com/nathansobo/node-plist.git",
+    "q": "~0.9.7",
+    "read": "~1.0.5",
+    "season": ">=5.0.5 <6.0",
+    "temp": "~0.7.0",
+    "underscore-plus": "1.x",
+    "wordwrap": "0.0.2",
+    "wrench": "~1.5.1",
+    "yargs": "^3.7.2"
```

Comment:
> Do we still want to support building with npm 1.x?

I think that is worth discussing, but probably in a separate pull request where all the dependencies are converted to use `^` instead of `>=X Y<` and `X.X` styles.